api/pubsub.py

```diff
@@ -3,40 +3,54 @@
 # Copyright (C) 2021 Collabora Limited
 # Author: Guillaume Tucker <guillaume.tucker@collabora.com>
 
-import aioredis
+"""Pub/Sub implementation"""
+
 import asyncio
+
+import aioredis
 from cloudevents.http import CloudEvent, to_json
 from pydantic import BaseModel, BaseSettings
 
 
 class Settings(BaseSettings):
+    """Pub/Sub settings loaded from the environment"""
     cloud_events_source: str = "https://api.kernelci.org/"
 
 
 class Subscription(BaseModel):
+    """Pub/Sub subscription object model"""
     id: int
     channel: str
 
 
 class PubSub:
+    """Pub/Sub implementation class"""
+
     ID_KEY = 'kernelci-api-pubsub-id'
 
     @classmethod
-    async def create(self, *args, **kwargs):
+    async def create(cls, *args, **kwargs):
+        """Create and return a PubSub object asynchronously"""
         pubsub = PubSub(*args, **kwargs)
         await pubsub._init_sub_id()
         return pubsub
 
-    def __init__(self, host='redis', db=1):
-        self._redis = aioredis.from_url(f'redis://{host}/{db}')
-        self._subscriptions = dict()
+    def __init__(self, host='redis', db_id=1):
+        self._redis = aioredis.from_url(f'redis://{host}/{db_id}')
+        self._subscriptions = {}
         self._lock = asyncio.Lock()
         self._settings = Settings()
 
     async def _init_sub_id(self):
         await self._redis.setnx(self.ID_KEY, 0)
 
     async def subscribe(self, channel):
+        """Subscribe to a Pub/Sub channel
+
+        Subscribe to a given channel and return a Subscription object
+        containing the subscription id which can then be used again in other
+        methods.
```

Comment:
I would suggest,
```     
        Subscribe to a given channel asynchronously and return a Subscription object
        containing the subscription id and channel which can then be used again in other
        methods.
```

Reply:
Only the id is needed for other methods in this class, which is why I didn't mention the channel explicitly.  It's part of the Subscription model, which is mentioned as the type of the returned object.